embassy-net-adin1110/src/lib.rs

```diff
@@ -0,0 +1,1249 @@
+#![allow(clippy::pedantic)]
+#![feature(type_alias_impl_trait)]
```

Comment:
`type_alias_impl_trait` is not actually used in the crate.

Reply:
Yes, you are right is wasn't needed. So it is removed.